src/Utils/ResponseConverter.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Resiliency\Utils;
+
+use Symfony\Contracts\HttpClient\ResponseInterface as SymfonyResponseInterface;
+use Psr\Http\Message\ResponseInterface;
+use Nyholm\Psr7\Response;
+
+/**
+ * Helper to convert SymfonyHttpClient Responses to PSR-7 Responses.
+ */
+final class ResponseConverter
+{
+    public static function convertToPsr7(SymfonyResponseInterface $response): ResponseInterface
+    {
+        return new Response();
```

Comment:
needs to be done and tested (this may be part of the SymfonyClient implementation)